glean-core/uniffi/src/private/recorded_experiment.rs

```diff
@@ -0,0 +1,16 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use std::collections::HashMap;
+
+use serde::Deserialize;
+
+/// Deserialized experiment data.
+#[derive(Clone, Deserialize, Debug)]
+pub struct RecordedExperiment {
+    /// The experiment's branch as set through [`set_experiment_active`](crate::set_experiment_active).
+    pub branch: String,
+    /// Any extra data associated with this experiment through [`set_experiment_active`](crate::set_experiment_active).
+    pub extra: HashMap<String, String>,
```

Comment:
No more `Option`. Breaks the API on the SDK side technically.

Reply:
With Nimbus as the only customer, I'm less concerned about breaking this particular API. I can deal with the changes downstream, if necessary.